### DISCUSSION TITLE
Using authme

### DISCUSSION THREAD

**Discussion by MBNAME12** [Levminer/authme#324] [Q&A]
I have downloaded authme to Linux and Windows platforms. I want to use authme to obtain secret keys from accounts and create TOTP codes for those accounts (manually entering secret keys). How do I do this, please ?

Also how do I save the TOTP codes and secret keys ?
Where are these codes and secret keys saved and are they automatically
encrypted ?

**Comment by Levminer** [comment]
1. You can import TOTP codes manually on the import page. TOTP > Enter a setup key
2. Codes are encrypted and saved inside localStorage.
